docs/source/gettingstarted.rst

```diff
@@ -162,21 +162,23 @@ We can also print the optimal value and solved variables individually.
 
 .. code-block:: python
 
-    print "The optimal value is %s." % (sol["cost"])
-    print "The x dimension is %s." % (sol(x))
+    print "The optimal value is %s." % sol["cost"]
+    print "The x dimension is %s." % sol(x)
+    print "The y dimension is %s." % sol["variables"]["y"]
 
 ::
 
     The optimal value is 15.5884619886.
-    The x dimension is 0.577351209028 meter.
+    The x dimension is 0.5774 meter.
+    The y dimension is 0.2887 meter.

```

Comment:
I think we do have that issue! Would be a great ticket for another documentation party...